packages/plugins/documentation/server/routes/index.js

```diff
@@ -80,4 +80,40 @@ module.exports = [
       policies: [],
     },
   },
+  {
+    method: "GET",
+    path: "/v:major(\\d+).:minor(\\d+).:patch(\\d+)/swagger.json",
+    handler: "documentation.getApiSpecJson",
+    config: {
+      auth: false,
+      middlewares: [restrictAccess],
+    },
+  },
+  {
+    method: "GET",
+    path: "/v:major(\\d+).:minor(\\d+).:patch(\\d+)/openapi.json",
+    handler: "documentation.getApiSpecJson",
+    config: {
+      auth: false,
+      middlewares: [restrictAccess],
+    },
+  },
+  {
+    method: "GET",
+    path: "/swagger.json",
+    handler: "documentation.getApiSpecJson",
+    config: {
+      auth: false,
+      middlewares: [restrictAccess],
+    },
+  },
+  {
+    method: "GET",
+    path: "/openapi.json",
+    handler: "documentation.getApiSpecJson",
+    config: {
+      auth: false,
+      middlewares: [restrictAccess],
+    },
+  },
```

Comment:
Cool thanks for this PR. Couldn't these routes be served up on the 'content-api'? The permissions to these endpoints could then be made public through the users permissions plugin or the endpoints could be accessed with an authenticated request. You could then also remove the config object on each route. 

You can see a similar setup on the i18n plugin: https://github.com/strapi/strapi/tree/master/packages/plugins/i18n/server/routes